src/components/NavBar.tsx

```diff
@@ -16,7 +16,7 @@ const sections = [
   },
   { id: 'screencasts', title: 'Screencasts' },
   { id: 'resources', title: 'Resources' },
-  { id: 'pricing', title: 'Pricing' },
```

Comment:
BTW, later it would be nice to change the id: to say "about-us" too, because that will be shown in the url (like "https://asteroid.org/#about-us"